sky/templates/aws-ray.yml.j2

```diff
@@ -76,23 +76,22 @@ initialization_commands: []
 # List of shell commands to run to set up nodes.
 setup_commands:
   # Make sure python3 & pip3 are available on this image.
-  # This also kills the service that is holding the lock on dpkg (problem only exists on aws/azure, not gcp)
   - pip3 --version > /dev/null 2>&1 || (curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py && echo "PATH=$HOME/.local/bin:$PATH" >> ~/.bashrc);
     (type -a python | grep -q python3) || echo 'alias python=python3' >> ~/.bashrc;
-  # We set auto_activate_base to be false for pre-installed conda.
-  # Refer to the issue link here: https://github.com/skypilot-org/skypilot/pull/1007
+  # patch the buggy ray files and enable `-o allow_other` option for `goofys`
+  # This also kills the service that is holding the lock on dpkg (problem only exists on aws/azure, not gcp)
```

Comment:
Seems to belong above L84.

Also, are the previous L82-83 ("We set auto_....") no longer applicable?